js/main.js

```diff
@@ -1,18 +1,79 @@
-function getRandomNumber(min, max) {
-  min = Math.ceil(min);
-  max = Math.floor(max);
-  return (min >= max || min < 0)
-    ? false
-    : Math.floor(Math.random() * (max - min + 1)) + min;
+function getRandomPositiveInteger (a, b) {
+  const lower = Math.ceil(Math.min(Math.abs(a), Math.abs(b)));
+  const upper = Math.floor(Math.max(Math.abs(a), Math.abs(b)));
+  const result = Math.random() * (upper - lower + 1) + lower;
+  return Math.floor(result);
 }
-getRandomNumber();
 
-function checkStringLength(stringCheck, maxSymbol) {
-  if (stringCheck.length <= maxSymbol) {
-    return true;
+(function checkStringLength (string, length) {
+  return string.length <= length;
+})('вызов функции для eslinter');
+
+const COMMENTATOR_NAMES = [
+  'Юлиан',
+  'Олег',
+  'Август',
+  'Влад',
+  'Нонна',
+  'Йоханна',
+  'Лидия',
+  'Генриетта',
+  'Елена',
+  'Евгений',
+  'Лоренс',
+  'Эрик',
+];
+
+const MESSAGE_EXAMPLES = [
+  'В целом всё неплохо. Но не всё.',
+  'Всё отлично!',
+  'Я поскользнулся на банановой кожуре и уронил фотоаппарат на кота и у меня получилась фотография лучше.',
+  'Лица у людей на фотке перекошены, как будто их избивают. Как можно было поймать такой неудачный момент?!',
+];
+
+function getRandomArrayElement (elements) {
+  return elements[getRandomPositiveInteger(0, elements.length - 1)];
+}
+
+function createNewNumericArray (min, max) {
+  const newArray = [];
+  for (let i = min; i <= max; i++) {
+    newArray.push(i);
+  }
+  return newArray;
+}
```

Comment:
Могу предложить упростить до
```suggestion
function createNumericArray(length = 10, start = 1) {
  return Array.from({ length }, (_, i) => start + i);
}
```

[Подробнее](https://stackoverflow.com/a/33352604)